packages/account/src/Configs/personal-details-config.ts

```diff
@@ -114,10 +116,10 @@ const personal_details_config = ({
             ],
         },
         tax_residence: {
-            default_value:
-                real_account_signup_target === 'maltainvest'
-                    ? account_settings.residence
-                    : residence_list.find(item => item.value === account_settings.tax_residence)?.text || '',
+            //if tax_residence is already set, we will use it as default value else for mf clients we will use residence as default value
+            default_value: account_settings?.tax_residence
+                ? residence_list.find(item => item.value === account_settings?.tax_residence)?.text
+                : default_residence,
```

Comment:
will it be safe for `default_value` to hold `undefined` when `residence_list.find(item => item.value === account_settings?.tax_residence)?.text` returns without any results?
shall we default it to an empty string (find || '')?